Source/Fuse.Triggers/Timeline.ScriptClass.uno

```diff
@@ -27,7 +27,7 @@ namespace Fuse.Triggers
 			
 			@scriptmethod pulse()
 		*/
-		static void pulse(Context c, Timeline n, object[] args)
+		static void pulse(Timeline n)
```

Comment:
What happens in the JS thread if the user calls these functions without arguments? They should get an error (as functions that take arguments can check and emit errors).

Reply:
Nothing, like before.

I can add a general error-case here if we really want to. So far I opted to try not to change any behavior in this PR.

Reply:
You can add this to the issue about converted the `args != 0` functions into zero-arg functions. I think it's quite important we start issuing errors on these. Allowing args hurts future compatibility.

Reply:
Yeah, good point. I've added that detail to the ticket.